R/html_dependency.R

```diff
@@ -154,32 +154,54 @@ validateScalarName <- function(x, name = deparse(substitute(x))) {
 #'
 #' @export
 htmlDependencies <- function(x) {
-  attr(x, "html_dependencies", TRUE)
+  unname(c(
```

Comment:
This is kind of hard to understand when all the logic is nested like this. Can you change it have multiple statements (and use variables)?